plugins/scaffolder-backend/config.d.ts

```diff
@@ -37,5 +39,23 @@ export interface Config {
      * Set to 0 to disable task workers altogether.
      */
     concurrentTasksLimit?: number;
+
+    /**
+     * Sets the processing interval for staled tasks.
+     *
+     * Staled tasks are marked as failed, once task's timeout has exceeded.
+     *
+     * By default, the frequency is every 5 minutes.
+     */
+    taskTimeoutReaperFrequency: HumanDuration;
```

Comment:
@freben I know that you implemented something like this before in another plugin but I can't remember. Was it called something like `janitor`? Should we keep the same pattern here rather than `reaper`?

Reply:
Maybe you're thinking about the one in the tasks package itself? https://github.com/backstage/backstage/blob/master/packages/backend-tasks/src/tasks/PluginTaskSchedulerJanitor.ts

Reply:
Would be nice if we come up with some naming convention and merge it, so that I can point to it in the doc for recoverable tasks: https://github.com/backstage/backstage/pull/20934/

Reply:
@acierto I'm thinking that we re-use the janitor naming like we've got in the `TaskScheduler`.

```
/**
 * Makes sure to auto-expire and clean up things that time out or for other
 * reasons should not be left lingering.
 */
```

Seems like a good fit for both right?

Reply:
@benjdlambert Sounds reasonable to me 👍  Updated the variable name and the description of it.